client/src/app/components/CreateProject/Form.js

```diff
@@ -36,17 +36,37 @@ class Form extends React.Component {
 
   onSubmit(e) {
     e.preventDefault();
-    console.log(this.state);
+
+    fetch("http://localhost:3000/projects/createProject", {
+      method: "POST",
```

Comment:
Why are you using `projects/createProject` instead of simply `projects` ?

Reply:
/projects in a client route, /projects/createProject is a server route

Reply:
Oh yes, I saw afterwards that you created a separate endpoint on the server, other than '/projects' POST. As you wish. I'd name it then /`projects/new` then, to avoid repetition of project